brave/brave-core
# Moved Adblock page to settings

Resolves https://github.com/brave/brave-browser/issues/8838

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Content filtering
Toggle on/off lists and see if it's being reflected in `brave://adblock` as it uses the same API. Also, search for a list to see if search input is functional

2. Add Custom Filter Lists
Test url: https://raw.githubusercontent.com/DandelionSprout/adfilt/master/Anti-'Custom%20cursors'%20List.txt
- Update the list, you will notice "Last updated" text change
- Remove the list
- Test with a non-filter url (google.com), you should be able to see "Download failure" as it's not a valid list

3. Custom Filters
Add CSS selectors and test if they're being blocked
